### DISCUSSION TITLE
How can I make the notes move around

### DISCUSSION THREAD

**Discussion by HahaYesLeLad** [KadeArchive/Kade-Engine#282] [Q&A] [answered]
how can I make the notes wave, and wiggle and shiz, like in the Vs Kris mod?

**Comment by DizzyJune** [comment]
Either two of these methods work

Easy method: Change the sprite sheet and the XML file. (Moves all notes in all songs with simple XML editing)

Hard method: Make new sprite sheets all with the notes in a different position and make a new XML that is only active during certain songs. (Requires more effort but is better overall)

**Comment by puyoxyz** [comment, marked as answer]
modcharts! https://github.com/KadeDev/Kade-Engine/blob/master/ModCharts.md

**Reply by HahaYesLeLad** [reply]
ty lmao
